cpp/src/table/row_operators.cu

```diff
@@ -393,6 +393,87 @@ namespace lexicographic {
 
 namespace {
 
+/**
+ * @brief Replace child of the input lists column by a new child column.
+ *
+ * If the input is not sliced, just replace the input child by the new_child.
+ * Otherwise, we have to generate new offsets and replace both offsets/child of the input by the
+ * new ones. This is because the new child was generated by ranking and always has zero
+ * offset thus cannot replace the input child if it is sliced.
```

Comment:
```suggestion
 * Otherwise, we have to generate new offsets and replace both offsets/child of the input by the
 * Otherwise, we have to generate new offsets and replace both the offsets and the child of the
 * input by the new ones. This is because the new child was generated by ranking and always
 * has zero offset, so it cannot replace the input child if it is sliced.
```